playbooks/deploy/deploy.yaml

```diff
@@ -4,15 +4,16 @@
     - quadlet
   become: true
   vars_files:
-    - "../vars/{{ certificate_source }}_certificates.yml"
-    - "../vars/images.yml"
+    - "../../vars/{{ certificate_source }}_certificates.yml"
+    - "../../vars/images.yml"
+    - "../../vars/database.yml"
   vars:
     certificate_source: default
     certificates_hostnames:
       - "{{ ansible_fqdn }}"
       - localhost
     certificates_ca_password: "CHANGEME"
-    candlepin_db_password: "CHANGEME"
+    candlepin_db_password: "{{ database_password }}"
```

Comment:
We should really discourage people from reusing database passwords between applications.